Confus/Confus/Networking/ClientConnection.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 #include <RakNet/MessageIdentifiers.h>
+#include <RakNet/RakPeerInterface.h>
```

Comment:
Wasn't the point of the forward declaration to move this to the cpp? Is there a reason for including this here as well?